File: codeeval/020--lowercase.js
/* Instructions */

/* Given a string write a program to convert it into lowercase.*/

/* Sample code to read in test cases: */
var fs  = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line != "") {
      line = line.toLowerCase();
      console.log(line);
    }
});

File: codeeval/020--lowercase.rb
# Instructions 

# Given a string write a program to convert it into lowercase.

# Sample code to read in test cases:

# HELLO CODEEVAL - hello codeeval
# This is some text - this is some text

# Sample code to read in test cases:
def lowercase(str)
  str.downcase
end

File.open(ARGV[0]).each_line do |line|
    puts lowercase(line)
end